extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -767,52 +743,36 @@ export class ProjectsController {
 		return extractTarget;
 	}
 
-	private async getFolderLocation(extractTarget: mssql.ExtractTarget): Promise<Uri | undefined> {
-		let selectionResult;
-		let projUri;
+	private async getFolderLocation(): Promise<Uri> {
+		let projUri: Uri;
 
-		if (extractTarget !== mssql.ExtractTarget.file) {
-			selectionResult = await this.apiWrapper.showOpenDialog({
-				canSelectFiles: false,
-				canSelectFolders: true,
-				canSelectMany: false,
-				openLabel: constants.selectString,
-				defaultUri: this.apiWrapper.workspaceFolders() ? (this.apiWrapper.workspaceFolders() as WorkspaceFolder[])[0].uri : undefined
-			});
-			if (selectionResult) {
-				projUri = (selectionResult as Uri[])[0];
-			}
-		} else {
-			// Get filename
-			selectionResult = await this.apiWrapper.showSaveDialog(
```

Comment:
just want to make sure - does import still work if the extract target is file? Previously, the file name was required if the extract target was file, but it looks like that was removed from getFolderLocation().

Reply:
Info in other reply